pkgs/development/python-modules/dataclasses-jsonschema/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, dataclasses
+, dateutil
+, jsonschema
+, typing-extensions
+, pytest
+, flake8
+, mypy
+, apispec
+, apispec-webframeworks
+, flask
+, pytest-runner
+, setuptools-scm
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dataclasses-jsonschema";
+  version = "2.14.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "153v7vkkhqhiz7mzxbslbgqp2z7xcgh9hvg3lyppjaigi3pa22sg";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace ", 'pytest-ordering'" ""
+  '';
+
+  nativeBuildInputs = [
+    setuptools-scm
+    pytest-runner
+  ];
+
+  propagatedBuildInputs = [
+    dateutil
+    jsonschema
+  ] ++ lib.optionals (pythonOlder "3.8") [ typing-extensions ]
+    ++ lib.optionals (pythonOlder "3.7") [ dataclasses ];
+
+  disabledTests = [
+    "embeddable"
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest
+    flake8
+    mypy
```

Comment:
```suggestion
```
pytest should not be duplicated and we do not execute lint tests.

Reply:
tests fail without mypy, the other two in fact appear to be unneeded.